tools/tests/add_and_update.py

```diff
@@ -0,0 +1,129 @@
+#! /usr/bin/env python3
+
+import sys
+
+from util import *
```

Comment:
add_and_update.py:5:0: W0614: Unused import(s) subprocess, yaml, TEST_PATH, APROJ_EXE, run_cmd, antler_proj_cmd, expected_cmd and expected_proj_cmd from wildcard import of util (unused-wildcard-import)
add_and_update.py:5:0: W0401: Wildcard import util (wildcard-import)